src/hooks/queries/usePreferencesQueries.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery, useMutation, useQueryClient } from '@tanstack/react-query';
+import { PreferencesService } from '@/services/PreferencesService';
```

Comment:
Add PreferencesService to index.ts and then the import looks like this:

```suggestion
import { PreferencesService } from '@/services';
```